### PR TITLE
Updated to Swift 5.

### PR DESCRIPTION
**Pull request by elland**

### CODE REVIEW

File: Teapot/Teapot+BasicAuth.swift [hunk 1]
```diff
@@ -8,7 +8,7 @@ extension String {
 
 public extension Teapot {
     /// The basic authentication header key value. Use this as the key in your headerFields dictionary.
-    public var basicAuthenticationHeaderKey: String {
+    var basicAuthenticationHeaderKey: String {
```
Comment (designatednerd) [inline Teapot/Teapot+BasicAuth.swift:11]: why does this no longer need to be public, because the extension itself is declared as public?

**Review by designatednerd — COMMENTED** [review]
One quick question on these changes. 

I'll open an issue to have a drag-out about `Result`

File: Teapot/Teapot+BasicAuth.swift (hunk 1 above)
Comment (elland) [reply Teapot/Teapot+BasicAuth.swift:11]: Precisely. Now they 'inherit' the access level.

**Comment by elland** [comment]
RE: Result:

We should def. re-implement it using the new `Result` type or at least explore it. 👍

**Comment by designatednerd** [comment]
@Elland See #40 for more feedback on that.

**Comment by designatednerd** [comment]
Do we not have CI set up for this?

**Comment by designatednerd** [comment]
Hm, looks like it got disconnected. @Elland can you push a dummy commit so this builds on Circle?

**Comment by elland** [comment]
Sure!

**Comment by elland** [comment]
Didn't work?

**Comment by designatednerd** [comment]
@Elland Can you try updating the `Xcode` in `.circleci/config.yml` to 10.2.0? I just replaced the webhook but I bet part of the issue is that.

Tests pass locally FWIW

**Comment by elland** [comment]
@designatednerd nothing?

**Comment by designatednerd** [comment]
I'm gonna merge this and fight with it on the `Result` PR.

**Pull request merged**